test/remote/gateways/remote_stripe_payment_intents_test.rb

```diff
@@ -10,6 +10,15 @@ def setup
     @declined_payment_method = 'pm_card_chargeDeclined'
     @three_ds_moto_enabled = 'pm_card_authenticationRequiredOnSetup'
     @three_ds_authentication_required = 'pm_card_authenticationRequired'
+    @three_ds_authentication_required_setup_for_off_session = 'pm_card_authenticationRequiredSetupForOffSession'
```

Comment:
Is this used anywhere?

Reply:
Updated test case to use both a cc and a pm_card.